### ISSUE TITLE
Add default value to dropout keep probability placeholder

### ISSUE THREAD

**Issue by geometrikal** [zsdonghao/tensorlayer#155]
In my Java app I load tensorflow graphs created using tensorlayer, that have been frozen into a pb file using the `freeze_graph.py` routine.

However, running the graph requires that all placeholders be fed a value, and this includes all the placeholders used for the keep probabilities. Since the graph can only be served, and not trained, in Java, it makes sense that these placeholders should default to a value of 1.0   

Therefore, is it possible to change the definition of the placeholder in the DropoutLayer from

    set_keep[name] = tf.placeholder(tf.float32)

to 

    set_keep[name] = tf.placeholder(tf.constant(1.0), [1], name=name+"/keep_prob")

This hopefully will default the keep probability to 1.0, give the node a name other than "Placeholder_X", and negate the need to feed the tensor when not training.

**Comment by zsdonghao** [comment]
Hi, you can set the dropout keep prob in `DropoutLayer` without placeholder, simply set `is_fix` to True, please check: https://github.com/wagamamaz/tensorlayer-tricks

**Comment by geometrikal** [comment]
Thanks, I think I understand. Your suggested method makes more sense when other layers that also need to be disabled are used.

**Comment by zsdonghao** [comment]
you are welcome
